tools/ci_build/build.py

```diff
@@ -99,12 +100,12 @@ def parse_arguments():
     parser.add_argument(
         "--enable_training_pipeline_e2e_tests", action="store_true",
         help="Enable the pipeline c++ e2e tests.")
-    parser.add_argument(
-        "--use_horovod", action='store_true', help="Enable Horovod.")
     parser.add_argument(
         "--mpi_home", help="Path to MPI installation dir")
     parser.add_argument(
         "--nccl_home", help="Path to NCCL installation dir")
+    parser.add_argument(
+        "--disable_mpi", action='store_true', help="Disable building wtih MPI.")
```

Comment:
Is there a reason to prefer --disable_mpi rather than having --use_mpi with a default to true?  [ In other settings we're generally using +ve flags ]

Reply:
I think it makes sense to change this to +ve wording as well.